productivity-suite/app/fragments/news/src/types.ts

```diff
@@ -0,0 +1,19 @@
+export interface IComment {
```

Comment:
I don't think that in typescript there is a convention to add an `I` in front of interface names, it feels strange to me and I haven't done this in the rest of the interfaces in the demo, what do we think? is there a real value in adding the `I` prefix?

Reply:
Copy pasted from the example repo

Reply:
I see, I am not a big fan but if we want to keep it as similar to the actual solid app as possible we could keep it like this...

I'd be curious to see if @petebacondarwin has any view on it 🙂

Reply:
FWIW I too hate the `IWhatever` convention, but I didn't want to spend time making the code match our conventions for the sake of consistency. I'm guessing Pete's out for the day and I don't want to hold this over the weekend over a single character, so I don't mind changing it.

That being said, I think it's better to keep the solid example close to the source material because:
- If people are actually following along and comparing this repo to the source hackernews repo, how will people know if we've changed something because it's necessary for piercing vs because we don't like the conventions.
- If we decide to update the hackernews code later, we'll have to re-apply all the tweaks we made. If we're only deleting code (but not modifying it) this is much easier to do. 
- The goal was to take an existing app in another framework and have it work with piercing. We're trying to showcase piercing, not show people how to build a solid app according to our standards.
- Having multiple apps with varying conventions and quality levels simulates a more real world scenario.

Reply:
Let's avoid changing the Solid example code if possible. I agree with the reasoning above.

Reply:
I also do not believe in prefixing interfaces with `I`.

Reply:
🥲👍